plugins/europeana/entity.js

```diff
@@ -182,3 +182,39 @@ function getRelatedEntityTitleLink(entities) {
   }
   return entityDetails;
 }
+
+/** Get the description for the entity
+ * If type is topic, use note
+ * If type is person, use biographicalInformation
+ * @return {String} a description when available in English
+ */
+export function getEntityDescription(type, entity) {
+  let description;
+
+  if (type === 'topic') {
+    description = entity.note && entity.note.en ? entity.note.en[0] : '';
+  } else if (type === 'person') {
+    if (entity.biographicalInformation.length !== undefined) {
```

Comment:
Can cause "Cannot read property 'length' of undefined" error, e.g. at /entity/person/18037-todd-woods